common/artifacts/000_wc_artifacts.txt

```diff
@@ -812,11 +1190,109 @@ chalice_of_trollbanes = {
 
 	health = 1
 
-	monthly_character_prestige = 1.5
+	monthly_character_prestige = 1
 
 	light_group_opinion = 5
 
 	picture = "GFX_inv_misc_trophy_argent"
+	slot = relic
+}
+orb_of_corruption = {
+	quality = 4
+	flags = { misc epic }
+
+	active = {
+		is_adult = yes
+	}
+
+	indestructible = yes
+
+	martial = 2
+	intrigue = 2
+
+	combat_rating = 10
+
+	monthly_character_prestige = 0.5
+	monthly_character_piety = 0.25
+
+	general_opinion = -10
+	fel_group_opinion = 20
+	death_god_opinion = 20
+
+	picture = "GFX_orb_of_corruption"
+	slot = relic
+}
+spellbook_of_drekthar = {
+	quality = 4
+	flags = { misc epic }
+
+	active = {
+		is_adult = yes			# maybe reduce requirements for shaman education?
+	
+		is_magic_class_trigger = yes	
+	}
+
+	stacking = no
+	indestructible = yes
+
+	diplomacy = 2
+	stewardship = 2
+	learning = 5
+	fertility = 0.1
+	health = 0.1
+
+	monthly_character_piety = 0.5
+
+	picture = "GFX_spellbook_of_drekthar"
+	slot = library
+}
+legion_doomhorn = {
+	quality = 3
+	flags = { misc rare }
+
+	active = {
+		is_adult = yes
+		OR = {
+			is_warlock_class_trigger = yes
+			religion = fel_group	
```

Comment:
Syntax error. `fel_group` is religion group and needs `religion_group` trigger.
```suggestion
			religion_group = fel_group	
```

Reply:
done